python/app/action_manager.py

```diff
@@ -191,13 +191,13 @@ def _get_default_detail_actions(self, sg_data):
         :param sg_data: Shotgun data directory
         """
         refresh = QtGui.QAction("Refresh", None)
-        refresh.triggered[()].connect(lambda f=sg_data: self._refresh(f))
+        refresh.triggered[()].connect(lambda: self._refresh(sg_data))
```

Comment:
Not only was this scoping unnecessary, but in PySide2+Python 3 we were receiving a boolean as a parameter, since triggered passes in a boolean, which broke all those callbacks.